pylintrc

```diff
@@ -150,6 +150,8 @@ disable=print-statement,
         # Ignore errors raised against `ExceptionSafeClass` and `ExceptionSafeAbstractClass`
         invalid-metaclass,
         broad-except,
+        no-name-in-module
+        import-error
```

Comment:
Added these to ignore the following lint errors:

```
mlflow/mleap.py (215,4): [E0611 no-name-in-module] No name 'version' in module 'mleap'
mlflow/mleap.py (215,4): [E0401 import-error] Unable to import 'mleap.version'
mlflow/mleap.py (216,4): [E0611 no-name-in-module] No name 'pyspark' in module 'mleap'
mlflow/mleap.py (216,4): [E0401 import-error] Unable to import 'mleap.pyspark.spark_support'
```

If `mleap.version` is not importable, an error should be thrown when `mlflow/mleap.py` is run.